RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETMemberType.cs

```diff
@@ -4,7 +4,6 @@ public enum CaDETMemberType
     {
         Method,
         Property,
-        Constructor,
-        Field
```

Comment:
Field is no longer belongs to Member ? This is reason why you make separate class CaDETField.cs ?

Reply:
Yes. In the previous version, all instances of CaDETMember that had the type field had most of their fields equal to null (as fields don't have accessedAccessors, invokedMethods and metrics, for example).

While this somewhat complicates the rest of the code (as we can't treat fields and other executable code (methods, properties, constructors) the same way, it should reduce confusion and bugs in the long run.